packages/manager/modules/pci/src/projects/project/training/jobs/info/info.html

```diff
@@ -279,8 +279,10 @@
                                 ng-repeat="data in $ctrl.job.spec.volumes track by $index"
                                 class="api-list-item"
                             >
-                                {{ data.container }}@{{
-                                data.region}}:{{data.mountPath}}:{{data.permission}}
+                                {{ data.container }}@{{data.region}}<span
+                                    ng-if="data.prefix !== ''"
+                                    >/{{data.prefix}}</span
+                                >:{{data.mountPath}}:{{data.permission}}
```

Comment:
Wouldn't it be easier to construct this string in a JS function ?

Reply:
Clearly, doing based on @mohammed-zahaf suggestion

Reply:
Clearly, will do based on @mohammed-zahaf suggestion